### PR TITLE
Add status endpoint

### LINKED ISSUE #7: Add plain /status endpoint
This endpoint returns a 200 OK which is the most basic check for knowing the service is running.

### PR DESCRIPTION
**Pull request by woutslakhorst**
- Added /status endpoint for basic health checking
- Added monitoring documentation

Fixes #7

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -21,27 +21,88 @@ The executable exposes the REST (or other) services from the different engines.
 .. image:: https://api.codacy.com/project/badge/Grade/272258ac93e847b9b61c08d4144d0538
     :target: https://www.codacy.com/app/woutslakhorst/nuts-go
 
-.. inclusion-marker-for-contribution
+Dependencies
+************
 
-Installation
-------------
+This projects is using go modules, so version > 1.12 is recommended. 1.10 would be a minimum.
```
Comment (stevenvegt) [inline README.rst]: Since we use some error handling of 1.13, should this be 1.13?

**Review by stevenvegt — COMMENTED** [review]
Should the status engine also be added to the list with engines so it availble during startup?

If this is an engine, shouln't we add it to the nuts-core so it can be used by other engines in their stand-alone mode?

File: README.rst (hunk 1 above)
Comment (woutslakhorst) [reply README.rst]: true

**Comment by woutslakhorst** [comment]
Logging and status are loaded implicitly, we could change this. But that is something apart from the current PR, don't you think

**Review by stevenvegt — APPROVED** [review]

**Pull request merged**